### ISSUE TITLE
melee kobolds

### ISSUE THREAD

**Issue by yrmvgh** [jeremygurr/dcssca#360]
01 kobold apts may need a slight tweak
01 when you make a kobold, and you look at the apts or the role/weapon recommendations
01 range seems like a good idea
01 and it is, but if you evolve a bunch of physical muts
02 you not only possibly lose your helmet, but you become a pretty beastly melee combatant
02 as such they might need some decent melee apts that aren't SB, so they have more options when the evolves tell them "time to go melee monster"!
04 I think the obvious choice is +1 LB, so that cross training can help you with the transition. but there may be a case for other weapons. maces have some excellent onehanders

**Comment by Sandman25DCSS** [comment]
Why? Being better at melee is actually a good reason to _decrease_ their aptitudes to compensate. We don't want to make Ko better than Tr, do we?

**Comment by yrmvgh** [comment]
> better than Tr

I may have made it sound better than it is when I used the term "melee monster". :stuck_out_tongue_closed_eyes:  Several rank 1 aux attacks that tend to fail against AC is not about to outperform a troll.

**Comment by Sandman25DCSS** [comment]
And my point is that lots of aux attacks make quickblade the best weapon :stuck_out_tongue_closed_eyes:

**Comment by yrmvgh** [comment]
Looking at monstrous kobolds from a completely different angle... you could argue that monstrous mutations make them more like demonspawn and less unique and "koboldy" ... whatever that means. Now that we have the "depends" tag, their niche could be acting as the less-melee-oriented, more-aux-armor-friendly, somewhat less-powerful-muts(on average) "Demonspawn Lite". Ds always get at least one brute mut, kobolds never do?

Basically what I'm saying is that maybe being a kobold isn't about having Spiny and Horns, it's about having Passive Mapping, Insight, +2 to a stat, regen 1, camouflage, shaggy fur, clarity, and a jelly that absorbs incoming projectiles... you dig?
